payments/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+
+# version: '4.15.0'
+# services:
+#   db:
+#     image: postgres:15.2
+#     restart: always
+#     environment:
+#       - POSTGRES_USER=tokyobanana
+#       - POSTGRES_PASSWORD=naganohara
+#     volumes:
+#       - db_data:/var/lib/postgresql/data
+#     ports:
+#       - '5432:5432'
+# volumes:
+#   db_data: {}
```

Comment:
Should remove unused code section. We already have the templates folder for skeleton so dont have to leave it here